test/function_wrappers.ts

```diff
@@ -105,4 +145,111 @@ describe('wrapUserFunction', () => {
     );
     func(request, response, () => {});
   });
+
+  describe('wraps a Typed JSON function', () => {
+    const synchronousJsonFunction: TypedHandlerFunction<
+      EchoMessage,
+      EchoMessage
+    > = {
+      format: new JsonInvocationFormat<EchoMessage, EchoMessage>(),
```

Comment:
is there a test that tests defaulting to the JsonInvocationFormat when no format is specified?

Reply:
Added a test that validates this